package.json

```diff
@@ -58,89 +58,91 @@
     "deploy-dev": "gh-pages -d styleguide -e dev -m \"[skip ci] Updates\""
   },
   "devDependencies": {
-    "@reach/rect": "0.11.2",
-    "@testing-library/jest-dom": "5.11.5",
-    "@testing-library/react": "11.1.1",
+    "@babel/core": "7.12.10",
+    "@reach/rect": "0.12.1",
+    "@testing-library/jest-dom": "5.11.6",
+    "@testing-library/react": "11.2.2",
     "@types/classnames": "2.2.11",
-    "@types/jest": "26.0.15",
+    "@types/jest": "26.0.19",
     "@types/jest-axe": "3.5.1",
-    "@types/react": "16.9.55",
-    "@types/react-dom": "16.9.9",
-    "@types/styled-components": "5.1.4",
+    "@types/react": "17.0.0",
+    "@types/react-dom": "17.0.0",
+    "@types/styled-components": "5.1.7",
     "@types/uuid": "8.3.0",
     "@types/warning": "3.0.0",
-    "@typescript-eslint/eslint-plugin": "4.6.1",
-    "@typescript-eslint/parser": "4.6.1",
-    "@wessberg/rollup-plugin-ts": "1.3.7",
-    "autoprefixer": "9.7.5",
+    "@typescript-eslint/eslint-plugin": "4.11.1",
+    "@typescript-eslint/parser": "4.11.1",
+    "@wessberg/rollup-plugin-ts": "1.3.8",
+    "autoprefixer": "10.1.0",
     "babel-jest": "26.6.3",
     "clean-css": "4.2.3",
-    "clipboard-copy": "3.2.0",
-    "cross-env": "7.0.2",
-    "css-loader": "5.0.0",
+    "clipboard-copy": "4.0.1",
+    "cross-env": "7.0.3",
+    "css-loader": "5.0.1",
     "cssnano": "4.1.10",
-    "eslint": "7.12.1",
-    "eslint-config-airbnb": "18.2.0",
-    "eslint-config-prettier": "6.15.0",
+    "eslint": "7.16.0",
+    "eslint-config-airbnb": "18.2.1",
+    "eslint-config-prettier": "7.1.0",
     "eslint-config-react-app": "6.0.0",
     "eslint-plugin-flowtype": "5.2.0",
     "eslint-plugin-import": "2.22.1",
-    "eslint-plugin-jest": "24.1.0",
+    "eslint-plugin-jest": "24.1.3",
     "eslint-plugin-jsx-a11y": "6.4.1",
-    "eslint-plugin-prettier": "3.1.4",
+    "eslint-plugin-prettier": "3.3.0",
     "eslint-plugin-react": "7.21.5",
     "eslint-plugin-react-hooks": "4.2.0",
     "eslint-plugin-tree-shaking": "1.8.0",
     "gh-pages": "3.1.0",
-    "husky": "4.3.0",
+    "husky": "4.3.6",
     "identity-obj-proxy": "3.0.0",
     "jest": "26.6.3",
     "jest-axe": "4.1.0",
     "jest-styled-components": "7.0.3",
-    "lint-staged": "10.5.1",
+    "lint-staged": "10.5.3",
     "node-sass": "5.0.0",
     "npm-run-all": "4.1.5",
-    "postcss-import": "12.0.1",
-    "prettier": "2.1.2",
+    "postcss": "8.2.1",
```

Comment:
` "postcss": "8.2.1",`
This seems to be totally new. Was something changed when updating that it is now required to have this dependency or was it just missing before?

Reply:
It was a missing peer dependency. Not sure if it was absolutely necessary to add it, but we do use it and if I'm correct, it's required to update the related postcss plugins (which were more than one major version behind latest already).